{
    "issue": {
        "title": "[Python] Fix warnings when running tests",
        "body": "***Note**: This issue was originally created as [ARROW-10281](https://issues.apache.org/jira/browse/ARROW-10281). Please see the [migration documentation](https://gist.github.com/toddfarmer/12aa88361532d21902818a6044fda4c3) for further details.*\n\n### Original Issue Description:\nWe have accumulated quite some warnings",
        "created_at": "2020-10-12T11:08:55.000Z",
        "updated_at": "2020-10-12T17:51:13.000Z",
        "labels": [
            "Migrated from Jira",
            "Component: Python",
            "Type: bug"
        ],
        "closed": true,
        "closed_at": "2020-10-12T17:51:06.000Z"
    },
    "comments": [
        {
            "created_at": "2020-10-12T17:51:06.538Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-10281?focusedCommentId=17212548) by Krisztian Szucs (kszucs):*\nIssue resolved by pull request 8439\n<https://github.com/apache/arrow/pull/8439>"
        }
    ]
}